Makefile

```diff
@@ -19,8 +19,8 @@ shell-check:
 VERSION:
 	tools/version.sh
 
-bump-python-version: VERSION
-	tools/bump-python-version.sh
+bump-python-version:
```

Comment:
Shouldn't this still depend on VERSION?

Reply:
Yup, but we should actually not bump the python version during the CI job but in the release commit/tag. I removed the target.